xbmc/cores/VideoPlayer/test/edl/TestEdl.cpp

```diff
@@ -458,8 +476,7 @@ TEST_F(TestEdl, TestMergeSmallCommbreaksAdvanced)
   // kodi should merge all commbreaks into two
   EXPECT_EQ(edl.GetEditList().size(), 2);
   // second edit of the original file + third one
-  EXPECT_EQ(edl.GetEditList().at(0).end - edl.GetEditList().at(0).start, (32 - 10) * 1000);
+  EXPECT_EQ(edl.GetEditList().at(0).end - edl.GetEditList().at(0).start, 32s - 10s);
   // 4th, 5th and 6th commbreaks joined
-  EXPECT_EQ(edl.GetEditList().at(1).end - edl.GetEditList().at(1).start,
-            std::lround((65.1 - 37) * 1000));
+  EXPECT_EQ(edl.GetEditList().at(1).end - edl.GetEditList().at(1).start, 65100ms - 37s);
```

Comment:
65.1s ?

Reply:
Same issue as above, avoiding the float on the seconds value